lms/djangoapps/instructor_task/api.py

```diff
@@ -333,7 +335,11 @@ def submit_calculate_problem_responses_csv(request, course_key, problem_location
     """
     task_type = 'problem_responses_csv'
     task_class = calculate_problem_responses_csv
-    task_input = {'problem_location': problem_location, 'user_id': request.user.pk}
+    task_input = {
+        'problem_location': problem_locations,
```

Comment:
The error is right here. This should be problem_locations. 
I'm afraid I changed this back to problem_locaiton for backwards compatibility but didn't update it in the final place. I will update it here now since the task name has already been changed to avoid clashes.